kpatch-build/kpatch-build

```diff
@@ -459,10 +481,15 @@ grep -q "CONFIG_DEBUG_KERNEL=y" "$OBJDIR/.config" || die "kernel doesn't have 'C
 if grep "CONFIG_LIVEPATCH=y" "$OBJDIR/.config" > /dev/null; then
 	# The kernel supports livepatch.
 	KBUILD_EXTRA_SYMBOLS=""
+	if version_gte ${ARCHVERSION//-*/} 4.7.0; then
+		USE_KLPRELAS=true
+		KPATCH_LDFLAGS="--unique=.parainstructions --unique=.altinstructions"
```

Comment:
The original rationale for this is the following:

If we have a patch that ends up generating multiple .parainstructions sections for different objects (e.g., kvm and vmlinux), these sections get merged into one big `.parainstructions` section when `output.o` is linked. However, these sections need to be applied per-object, with separate `.klp.arch.$objname.` sections. It seemed easier to pass in `--unique` than to have `create-klp-module` (which is invoked after the patch module .ko is built) iterate through a merged `.parainstructions` section and tease out which parts of it belong to what object.

But looking at this again, I think this does the wrong thing when we deal with multiple .o's from the same object that have their .parainstructions sections included. The upstream livepatch code doesn't handle the case when a single object has multiple .klp.arch. sections. :-/ 

So maybe the correct thing to do is to not use --unique and have `create-klp-module` do the harder thing (which is to iterate through the arch sections and manually separate them per-object...)